src/tests/modules/lua/mod1.lua

```diff
@@ -1,8 +1,8 @@
 function authorize()
 	if not type(fr) == "table" then
 		print("error: the 'fr' should be a table")
-		return fr.fail
+		return "fail"
 	end
 
-	return fr.ok
+	return "noop"
```

Comment:
tests that pass should be `ok` not `noop`

Reply:
fixed.